src/main/java/uk/co/onsdigital/discovery/model/DimensionalDataSet.java

```diff
@@ -20,11 +20,16 @@
 )
 @NamedQueries({
         @NamedQuery(name = "DimensionalDataSet.findAll", query = "SELECT d FROM DimensionalDataSet d ORDER BY d.s3URL"),
-        @NamedQuery(name = "DimensionalDataSet.count", query = "SELECT COUNT(d) FROM DimensionalDataSet d")
+        @NamedQuery(name = "DimensionalDataSet.count", query = "SELECT COUNT(d) FROM DimensionalDataSet d"),
+        @NamedQuery(name = DimensionalDataSet.FIND_BY_EDITION_VERSION,
+                    query = "SELECT d FROM DimensionalDataSet d WHERE d.major_label = :edition AND d.minor_version = :version ORDER BY d.s3URL"
+        )
 })
 public class DimensionalDataSet implements Serializable {
     private static final long serialVersionUID = 1L;
 
+    public static final String FIND_BY_EDITION_VERSION = "DimensionalDataSet.findByEditionVersion";
```

Comment:
I think we need constants for the parameters in the query too, especially as they're not the same names as the fields being queried.